app/tag-management/all-tags/tag-list/tag-list.component.html

```diff
@@ -0,0 +1,22 @@
+<ol class="tag-list">
+  <li *ngFor="let tag of tags; let isLast = last">
+    <div class="tag-header">
+      <h3>{{ tag.name }}</h3>
```

Comment:
I don't think that a tag name is a headline in this situation. use a span or p instead.